vectordatasource/transform.py

```diff
@@ -8959,23 +8959,38 @@ def unpack_viewpoint_claims(shape, props, fid, zoom):
     For example; "claimed_by=AA,BB,CC" should become "kind:aa=country,
     kind:bb=country, kind:cc=country" (or region, etc... as appropriate for
     the main kind, which should be "unrecognized_TYPE".
+
+    Additionally, "recognized_by=XX,YY,ZZ" indicates that these viewpoints,
```

Comment:
Most the data is `;` delimitated, should our examples (here and above) be as well?

Reply:
Good point! I switched it to `;` in 2fa74855.